deploy-manage/users-roles/cluster-or-deployment-auth/orchestrator-managed-users-overview.md

```diff
@@ -0,0 +1,27 @@
+---
+applies_to:
+  deployment:
+    ess:
+    ece:
+    eck:
+navigation_title: Orchestrator-managed users (elastic user)
+---
+
+# Orchestrator-managed users (elastic user)
+
+The {{es}} provides user credentials to help you get up and running. This includes credentials for the `elastic` user.
+
+In self-managed clusters, these users are created as [built-in users](/deploy-manage/users-roles/cluster-or-deployment-auth/built-in-users.md).
+
+In orchestrated deployments (ECH, ECE, and ECK), these are users in the [file realm](/deploy-manage/users-roles/cluster-or-deployment-auth/file-based.md). The way that credentials are managed for this user depends on your orchestrator.
+
```

Comment:
I wouldn't explicitly mention or link the file realm here, as I don't think we have in our public docs the details on how these users are implemented. Technically they are not elasticsearch built-in users but the way they are implemented should probably be out of the scope of the doc. Let me suggest a small change, but feel free to do it in your own way.

```suggestion
In orchestrated deployments (ECH, ECE, and ECK), the `elastic` user is managed by the platform, while other built-in users are not accessible to end users. The way that credentials are managed for this and other internal users depends on your orchestrator.
```